packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -1,252 +1,194 @@
-
-      /* stylelint-disable */
-      @media (-ms-high-contrast: active), (forced-colors: active) {
-        border-color: ButtonText;
-      }
-      /* stylelint-enable */
+      opacity: 1;
     }
 
-    &::after {
-      opacity: 1;
+    &:focus {
+      border-color: $color-primary-darker;
     }
   }
 
-  // Disabled choice
-  &:disabled + label {
+  // Disabled
+  &:disabled {
+    background-color: $choice-disabled-background-color;
+    border-color: $choice-disabled-border-color;
     color: $color-muted;
     cursor: not-allowed;
-
-    &::before {
-      background-color: $choice-disabled-background-color;
-      border-color: $choice-disabled-border-color;
-    }
-
-    /* stylelint-disable */
-    @media (-ms-high-contrast: active), (forced-colors: active) {
-      color: GrayText;
-
-      &::before {
-        background-color: Window;
-        border-color: GrayText;
-      }
-    }
-    /* stylelint-enable */
   }
 
-  // Focused choice
-  &:focus + label::before {
+  // Focus
+  &:focus {
     @include focus-styles;
   }
 }
 
-.ds-c-choice--inverse {
-  // Choice checkbox/radio button
-  + label::before {
-    background-color: $choice-background-color-inverse;
-    border-color: $choice-border-color-inverse;
+// Checkbox styles
+.ds-c-choice[type='checkbox'] {
+  &::after {
+    background-color: transparent;
+    border: solid;
+    border-color: $color-white;
+    border-top-color: transparent;
+    border-width: 0 0 4px 4px;
+    content: '';
+    grid-area: 1/-1;
+    height: 4px;
+    opacity: 0;
+    transform: rotate(-45deg) translateX(0.075em);
+    width: 10px;
   }
 
-  // Errored inverse choice
-  @if $ds-include-choice-error-highlight {
-    &.ds-c-choice--error + label::before {
-      border-color: $color-error-light;
+  &:checked {
+    &::after {
+      opacity: 1;
     }
+
+    /* stylelint-disable max-nesting-depth, selector-max-specificity */
+    &:focus::before {
+      height: $choice-size - ($choice-border-width * 2);
+      width: $choice-size - ($choice-border-width * 2);
+    }
+    /* stylelint-enable max-nesting-depth, selector-max-specificity */
   }
+}
 
-  // Checked inverse choice
-  // Need to repeat this styling to override error border-color when checked
-  &:checked + label::before {
-    border-color: $choice-checked-background-color;
+// Radio styles
+.ds-c-choice[type='radio'] {
+  border-radius: 100%;
+
+  &::before {
+    border-radius: 100%;
+    height: $choice-radio-button-size;
+    width: $choice-radio-button-size;
   }
+}
 
-  // Disabled inverse choice
-  &:disabled + label {
-    color: $color-muted-inverse;
+.ds-c-choice--inverse {
+  background-color: $choice-background-color-inverse;
+  border-color: $choice-border-color-inverse;
 
-    &::before {
-      background-color: $choice-disabled-background-color-inverse;
-      border-color: $choice-disabled-border-color-inverse;
+  // Error
+  @if $ds-include-choice-error-highlight {
```

Comment:
This is interesting, I didn't realize we were using style toggles like this.. Feels a little awkward, like maybe anywhere this is used it should just be another style? 

Wondering about working these into theming now. It looks like this is the only variable that I can see used in this way.

Reply:
I agree. I sent a message to [the CMS Slack](https://cmsgov.slack.com/archives/CT169N26L/p1648567752752839) for more guidance since I don't know what purpose this serves.